.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: generic
+
+env:
+  - DOCKER_COMPOSE_VERSION=1.14.0
+
+install:
+  # Installing docker
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
+
+  # Installing docker-compose
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+before_script:
+  # Check docker & docker-compose versions
+  - docker --version
+  - docker-compose --version
+
+script:
+  - docker-compose build
+  - docker-compose up -d
+
+  # Verifications
+  - sleep 4m
```

Comment:
4 minutes to wait for the stack to be ready? I reckon `1m` should be enough.

Reply:
In the past I've tryed with lower values than ``4m``, like in this case (``3m``) :
https://travis-ci.org/ivanrome/docker-elk/builds/286973782

Note the failure of the build due to the error connecting with kibana
> The command "curl --retry 10 --retry-delay 5 -v http://localhost:5601/" exited with 52.
Note that in the past ``3m`` was enough time to start the container but not always. So I increased the value to ``4``

Reply:
Ok, I forgot that it needs to build all the images first. Maybe upgrade that to `5m` then.

Reply:
Won't be necessary because the images are already built when `sleep` is executed.

Reply:
Yeah, exactly, images are built before starting the container. What it takes time is starting the stack

Reply:
OK 2f58ba5f6157dcf21df49430eaad8e9850dca60d